tests/common/output.rs

```diff
@@ -1,13 +1,83 @@
-const FIX_SENTENCE: &str = "All warnings are fixed. Total:";
+enum Mode {
```

Comment:
You don't need to create this `enum` - please use already defined.

Reply:
I did this to avoid having to make the `common` module (as well as `common/output`) public in the crate. Is it okay to make them public?

Reply:
Let's leave it as it is.